armi/reactor/tests/test_blocks.py

```diff
@@ -1804,14 +1829,31 @@ def test_rotateBoundaryParameters(self):
 
     def _rotateAndCompareBoundaryParams(self, rotNum: int, expected: np.ndarray):
         self.block._rotateBoundaryParameters(rotNum)
-        for name in self.ROTATABLE_BOUNDARY_PARAMS:
+        for name in self.BOUNDARY_PARAMS:
             data = self.block.p[name]
             msg = f"{name=} :: {rotNum=} :: {data=}"
             np.testing.assert_array_equal(data, expected, err_msg=msg)
-        for name in self.STATIC_BOUNDARY_PARAMS:
-            data = self.block.p[name]
-            msg = f"{name=} :: {data=}"
-            np.testing.assert_array_equal(data, self.BOUNDARY_DATA, err_msg=msg)
+
+    def test_rotatedPinParameters(self):
+        """Test that block parameters that reflect pin data are rotated.
+
+        Pre-rotate pin layout -> Post-rotate layout
+
+              2  1      1  6
+            3  0  6 -> 2  0  5
+              4  5      3  4
+
+        - Pre-rotate data: ``[0, 1, 2, 3, 4, 5, 6, ...]``
+        - Post-rotate data: ``[0, 6, 1, 2, 3, 4, 5, ...]``
+
```

Comment:
```suggestion
        Pre-rotate pin layout -> Post-rotate layout::

              2  1      1  6
            3  0  6 -> 2  0  5
              4  5      3  4

        - Pre-rotate data: ``[0, 1, 2, 3, 4, 5, 6, ...]``
        - Post-rotate data: ``[0, 6, 1, 2, 3, 4, 5, ...]``
```

Reply:
I don't think your diagram will render in RST without the `::`.  Tell me if I'm wrong.

Reply:
Yeah you're right. I wasn't sure we included test docstrings in the docs but we do so I'll fix it